recipes/cyclus/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "$(uname)" == "Darwin" ]; then
+  # toolchain copy
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.9"
+  export MACOSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+  export CMAKE_OSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
+  export CFLAGS="${CFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
+  export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -lc++"
+  export LINKFLAGS="${LDFLAGS}"
+  export CFLAGS="${CFLAGS} -m${ARCH}"
+  export CXXFLAGS="${CXXFLAGS} -m${ARCH}"
```

Comment:
Ideally we wouldn't copy the `toolchain`, but I'm not going to worry about it. Getting this working has already take quite a bit of work.